lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -922,8 +922,9 @@ WitnessChecker::lookupValueWitnessesViaImplementsAttr(
   }
 }
 
-SmallVector<ValueDecl *, 4> 
-WitnessChecker::lookupValueWitnesses(ValueDecl *req, bool *ignoringNames) {
+SmallVector<ValueDecl *, 4>
+WitnessChecker::lookupValueWitnesses(ValueDecl *req, bool *ignoringNames,
+                                     NormalProtocolConformance *conformance) {
```

Comment:
Passing the conformance down feels like a hack. Is there no other way to get the type you need?